app/models/deidentified_client.rb

```diff
@@ -7,4 +7,77 @@ class DeidentifiedClient < NonHmisClient
     text.strip!
     where(search_alternate_name(text))
   end
+
+  def download_headers
```

Comment:
I think we have used a hash to combine this and `download_data` elsewhere?

Reply:
We have (at least in the warehouse.)   At this time this is only for the export, so it seemed somewhat less necessary to build all of the hashes.  That said, it would make it less error prone.